Source/DafnyLanguageServer/Workspace/DafnyDocument.cs

```diff
@@ -24,6 +24,7 @@ public record DafnyDocument(
     DafnyOptions Options,
     TextDocumentItem TextDocumentItem,
     IReadOnlyList<Diagnostic> ParseAndResolutionDiagnostics,
+    IReadOnlyList<IImplementationTask> VerificationTasks,
```

Comment:
> Otherwise, future work that will hook into the VerificationTasks will have to do again the work of regrouping tasks per methods.

Why not make that change then?

Reply:
That's right, since DafnyDocument is private, we can do this then.